src/billing/constants/index.ts

```diff
@@ -291,7 +291,6 @@ export const countries = [
   'Ukraine',
   'United Arab Emirates',
   'United Kingdom',
-  'United States',
```

Comment:
you should add this back. for when a person misses that it's the first country, and does the logical thing of searching alphabetically, or typing in `unite`

Reply:
I don’t know if this is typable but that’s a good point

Reply:
I tried changing it. It started giving duplicate key error. Looked at clockface and it turns out `SelectDropdown` doesn't support duplicate values in it.

Reply:
maybe the problem is with the first one and not this one then. instead of autoselecting `myarray[0]`, make it auto select `united states` until we make that reflect the region they're currently in (fancy)

Reply:
makes sense. will take care of it tomorrow